app/models/project.rb

```diff
@@ -31,6 +31,11 @@ class Project < ActiveRecord::Base
     end
   end
 
+  def self.selected
+    project_names = Team.in_current_season.accepted.pluck(:project_name).uniq
```

Comment:
Let's skip the `uniq` here - it will convert the whole thing to an Array (and actually execute the query), which is not really necessary (there can be loads of duplicates, the final resulting Projects will be the same :v:)